daprdocs/content/en/developing-applications/building-blocks/workflow/howto-author-workflow.md

```diff
@@ -28,19 +28,86 @@ The Dapr sidecar doesn’t load any workflow definitions. Rather, the sidecar si
 
 ## Write the workflow activities
 
-Define the workflow activities you'd like your workflow to perform. Activities are a class definition and can take inputs and outputs. Activities also participate in dependency injection, like binding to a Dapr client.
+Define the workflow activities you'd like your workflow to perform. Activities are a class definition and can take inputs and outputs. Activities also participate in dependency injection, like binding to a Dapr client. 
```

Comment:
Does this paragraph apply to all SDKs? Go is not class orientated and will Go/JS/Python SDKs have the concept of dependency injection? Asking since this may be best moved into the .NET section and a more general statement added for what an activity is.